### DISCUSSION TITLE
How to swap x-axis and y-axis in Axes?

### DISCUSSION THREAD

**Discussion by wangjia184** [ManimCommunity/manim#3581] [Q&A]
I tried rotate, it does not work.

Can that be done?

Thanks in advance

**Comment by uwezi** [comment]
Without any code or explanation what you have tried and failed it's difficult to help you. Come over to Discord, open a topic in the help forum and describe exactly what you want.
[Discord: where can I find more resources for learning manim?](https://docs.manim.community/en/stable/faq/general.html?highlight=discord#where-can-i-find-more-resources-for-learning-manim)
